schema/project-level/project-schema.json

```
@@ -258,7 +258,7 @@
         },
         "description": {
           "title": "Description",
-          "description": "The description should summarise the purpose of this contract and the scope of work to be undertaken.",
+          "description": "The description should summarise the purpose of this contract and the initial scope of work to be undertaken.",
```

Somehow I think this makes it clearer that we mean the initial scope for the whole contract, rather than the scope of the initial work that will happen under the contract:

```suggestion
          "description": "The description should summarise the purpose of this contract and the initial scope of the work to be carried out under the contract.",
```